src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,21 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            if (false !== $this->options['differ']) {
+                // 'diff' value does not matter here
```

Comment:
>One could expect that changing the diff type does not implicitly enables diff output.

I doubt that, why change or set a format of something you don't want?

I'm not going back and forward on copy and branch change requests that were not a problem before unless someone points out what relevant changes have happened between than and now.